src/avram/slugify.cr

```diff
@@ -49,6 +64,22 @@ module Avram::Slugify
     end
   end
 
+  def generate(using slug_candidate : Avram::Attribute(String) | String) : String
+    generate([slug_candidate])
+  end
+
+  def generate(using slug_candidates : Array(String | Avram::Attribute(String) | Array(Avram::Attribute(String)))) : String
+    slug_candidates = format_candidates(slug_candidates)
+
+    slug_candidates.first? || UUID.random.to_s
+  end
+
+  private def format_candidates(slug_candidates : Array(String | Avram::Attribute(String) | Array(Avram::Attribute(String)))) : Array(String)
+    slug_candidates.map do |candidate|
+      parameterize(candidate)
+    end.reject(&.blank?)
```

Comment:
Should this include a call to [`#strip`](https://crystal-lang.org/api/1.2.2/String.html#strip%3AString-instance-method) as well?